webrtc-stats.html

```diff
@@ -632,12 +632,12 @@
                 </p>
               </dd>
               <dt>
-                <dfn><code>parameters</code></dfn> of type <span class=
+                <dfn><code>sdpFmtpLine</code></dfn> of type <span class=
                 "idlMemberType"><a>DOMString</a></span>
               </dt>
               <dd>
                 <p>
-                  From the SDP description line.
+                  The a=fmtp line in the SDP corresponding to the codec, as defined by [[!JSEP]] (section 5.7.).
```

Comment:
Need to say where in the a=fmtp line this starts (after the colon following the PT).